tests/common/devices.py

```diff
@@ -1138,6 +1138,22 @@ def get_asic_name(self):
 
         return asic
 
+    def get_running_config_facts(self):
+        return self.config_facts(host=self.hostname, source='running')['ansible_facts']
+
+    def get_vlan_intfs(self):
+        '''
+        Get any interfaces belonging to a VLAN
+        '''
+        vlan_members_facts = self.get_running_config_facts()['VLAN_MEMBER']
```

Comment:
For non-t0 topology, key 'VLAN_MEMBER' would not exist in config_facts. Exception 'KeyError' could be raised here.
Suggest to use code like:
```
        vlan_members_facts = self.get_running_config_facts().get('VLAN_MEMBER', {})
```

Reply:
Good point, I will make this change in a future PR